pkg/kubectl/autoscale.go

```diff
@@ -20,10 +20,9 @@ import (
 	"fmt"
 	"strconv"
 
+	autoscaling "k8s.io/api/autoscaling/v1"
```

Comment:
We should pick the most stable one that we can expect to be available.  E.g. when a new stable API comes out, we don't want to immediately change to use that API since it won't work with skewed clusters.  We *could* support check for which API is available, and using the most mature version.  Adds some complexity though.